main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/model/parameter/SettingVisitor.java

```
@@ -507,10 +507,13 @@ private void buildActivationCondition(final PropertyNode node, final PropertyNod
 
         node.getProperty().getCondition()
                 .forEach(c -> {
-                    c.setTargetPath(pathResolver.resolvePath(node.getProperty().getPath(), c.getTarget()));
-                    activations.computeIfAbsent(origin.getProperty().getPath(), (key) -> new HashMap<>());
-                    activations.get(origin.getProperty().getPath()).computeIfAbsent(level, (k) -> new ArrayList<>());
-                    activations.get(origin.getProperty().getPath()).get(level).add(c);
+                    final String targetPath = pathResolver.resolvePath(node.getProperty().getPath(), c.getTarget());
+                    if (targetPath != null && !targetPath.isEmpty()) {
+                        c.setTargetPath(targetPath);
+                        activations.computeIfAbsent(origin.getProperty().getPath(), (key) -> new HashMap<>());
+                        activations.get(origin.getProperty().getPath()).computeIfAbsent(level, (k) -> new ArrayList<>());
+                        activations.get(origin.getProperty().getPath()).get(level).add(c);
+                    }
```

Not sure it is ok to ignore unresolved parameter silently.
At least it is worth to log it in Debug mode

---

I will add an error log here, but, i'm for a fail fast paradigm. 
Here, it's to late to catch this kind of errors. We may add some more validations to the component model (maven plugin and container creation)